.github/workflows/continuous-integration-windows.yml

```
@@ -31,8 +31,8 @@ jobs:
         cmake --build build --parallel 4 --config Release
   windows-serial:
     # Serial build on Windows
-    name: Serial VS-2019
-    runs-on: windows-2019
+    name: Serial VS-2022
+    runs-on: windows-2022
```

I'd just switch to using the latest version since that's likely most relevant going forward.
```suggestion
    name: Serial VS-2025
    runs-on: windows-2025
```
Not sure if we should raise the minimum version as well.

---

Given how picky msvc is I am for bumping high. But a lot of windows projects I have seen are rather slow upgrading ...

---

The minimum for C++20 is windows 2022 already so we might be fine.

---

Let's do 25 here, and leave the MSVC+CUDA build at 22.

---

The compiler version does not seem to be any different between the two builds:
```
-- Building for: Visual Studio 17 2022
-- Setting default Kokkos CXX standard to 17
-- The CUDA compiler identification is NVIDIA 12.4.131 with host compiler MSVC 19.43.34808.0
-- The CXX compiler identification is MSVC 19.43.34808.0
```
vs.
```
-- Building for: Visual Studio 17 2022
-- Setting default Kokkos CXX standard to 17
-- The CXX compiler identification is MSVC 19.43.34808.0
```

---

its windows, who knows ... 
But in this case, I think it is definitely reasonable to go to 2025